src/pages/ItemDetail/ItemDetail.jsx

```diff
@@ -1,7 +1,216 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './ItemDetail.scss';
+import ShoesSize from './components/ShoesSize/ShoesSize';
+import DetailImgs from './components/DetailImgs/DetailImgs';
+import Counter from './components/Counter/Counter';
+import Modal from './components/Modal/Modal';
+import ShoesColor from './components/ShoesColor/ShoesColor';
+import ShoesModal from './components/ShoesModal/ShoesModal';
+import Review from './components/Review/Review';
+import { useParams } from 'react-router-dom';
+// import ShoesModal from './components/ShoesModal/ShoesModal';
+/* eslint-disable */
 
 function ItemDetail() {
-  return <h1>ItemDetail</h1>;
+  const [modal, setModal] = useState(false);
+  const [product, setProduct] = useState({});
+  const [shooseSize, setShooseSize] = useState('');
+  const [shoesModal, setShoesModal] = useState(false);
+  const [reviewOpen, setReviewOpen] = useState(false);
+  const [quantity, setquantity] = useState(1);
+
+  const params = useParams();
+  console.log(params);
+  // const [discount, setDiscount] = useState('');
+
+  const openModal = () => {
+    setModal(prev => !prev);
+    document.body.style.overflow = 'hidden';
+    window.scroll(0, 15);
+  };
+  const closeModal = () => {
+    setModal(prev => !prev);
+    document.body.style.overflow = 'unset';
+  };
+
+  const openShoesModal = () => {
+    setShoesModal(prev => !prev);
+    document.body.style.overflow = 'hidden';
+  };
+  const closeShoesModal = () => {
+    setShoesModal(prev => !prev);
+    document.body.style.overflow = 'unset';
+  };
+
+  const openReview = () => {
+    setReviewOpen(prev => !prev);
+  };
+
+  const onIncrease = () => {
+    setquantity(prevquantity => prevquantity + 1);
+  };
+  const onDecrease = () => {
+    setquantity(prevquantity => prevquantity - 1);
+  };
+
+  // const params = usePrams();
+  // const userId = params.id;
+
+  // const [user, setUser] = useState();
+
+  useEffect(() => {
+    fetch('./data/Mock.json')
+      .then(res => res.json())
+      .then(data => setProduct(...data));
+  }, []);
+
+  return (
+    <section className="ItemDetail">
```

Comment:
```suggestion
    <section className="itemDetail">
```
가장 최상위 부모태그에 컴포넌트 이름과 동일하게 잘 작성해 주셨는데
className은 첫글자가 소문자로 시작하는 카멜 케이스여야 합니다!